ExomeDepth/database/functions.py

```diff
@@ -126,3 +127,72 @@ def get_sample_id(bam):
         sampleid = list(set(sampleid))
         sampleid = "_".join(sampleid)
     return sampleid
+
+
+def get_folder_sorted(path):
+    folders = sorted(set(glob.glob("{}*".format(path), recursive=True)))
+    return folders
+
+
+def get_qc_bam_files(folder):
+    bam_files = []
+    qc_file = glob.glob("{}/QC/CNV/*exomedepth_summary.txt".format(folder), recursive=True)
+    if not qc_file or len(qc_file) > 1:
+        print("WARNING: CNV QC file missing of multiple detected in folder {} Skipping all samples in folder!".format(folder))
+        return None, None, True
+
+    for nf_bam in glob.glob("{}/bam_files/*.bam".format(folder), recursive=True):  # Nextflow analysis
+        bam_files.append(nf_bam)
+
+    for iap_bam in glob.glob("{}/*/mapping/*realigned.bam".format(folder), recursive=True):  # IAP analysis
+        bam_files.append(iap_bam)
```

Comment:
Combining could result in 'legacy problems'  as there could be other BAM files/folders (i.e. folders in old Exomedepth analysis). Fixing these exceptions is possbile, but I guess this would make it even more complicated.
I prefer to leave it as this.

f-string is a good suggestion. Changed.